packages/flutter/lib/src/widgets/image.dart

```diff
@@ -1100,6 +1119,15 @@ class _ImageState extends State<Image> with WidgetsBindingObserver {
   StackTrace? _lastStack;
   ImageStreamCompleterHandle? _completerHandle;
 
+  /// True when animations are disabled and the image should not update, such as
+  /// when [TickerMode] is disabled or [MediaQueryData.disableAnimations] is
+  /// true.
+  bool _isPaused = false;
+
+  /// False when the class first is instantiated and true forever after the
+  /// first frame of the image is received.
```

Comment:
What if image source changes?

Reply:
I think you're right, if the image source changes we  want to load the first frame of the new image. I'll do that.